docs/API.md

```diff
@@ -728,6 +728,18 @@ const Counter = componentFromStream(props$ => {
 })
 ```
 
+### `mapPropsStreamWithConfig()`
```

Comment:
It's really bad for libraries though. We should at least note not to use it if you're using it to create a library.